mw-webapp/src/logic/wayPage/reportsTable/modalContentJobTags/ModalContentJobTags.tsx

```diff
@@ -44,9 +46,18 @@ interface JobDoneTagsProps {
 export const ModalContentJobTags = (props: JobDoneTagsProps) => {
   const [jobTagsUpdated, setJobTagsUpdated] = useState<string[]>(props.jobDoneTags);
 
-  const uniqueJobTags = Array.from(new Set(jobTagsUpdated));
+  const isJobTagsEmpty = jobTagsUpdated.length === 0;
+  const isJobTagsSingle = jobTagsUpdated.length === DEFAULT_AMOUNT_TAGS;
+
+  const filteredJobTags = isJobTagsSingle
+    ? jobTagsUpdated
+    : Array.from(new Set(jobTagsUpdated)).filter((tag) => tag !== "no tag");
+
+  const checkedJobTags = isJobTagsEmpty
+    ? jobTagsUpdated.concat("no tag")
+    : filteredJobTags;
 
-  const allTags = Array.from(new Set(props.jobTags.concat(props.jobDoneTags).filter((tag) => tag !== DEFAULT_TAG)));
+  const allTags = Array.from(new Set(props.jobTags.concat(checkedJobTags).filter((tag) => tag !== DEFAULT_TAG)));
```

Comment:
Why do we need set here?

Reply:
Because we dont want to see tags duplicates

Reply:
Whu should we have duplicates?

Reply:
Because props.jobTags is all tags on the way and checkedJobTags is tags that was used on special joDone or plan. So all jobsDoneTags includes checkedJobTags by default, but we don't know which one. So for delete duplicate we use Set